kratos/python_scripts/assign_mpcs_to_neighbours_process.py

```diff
@@ -0,0 +1,178 @@
+import KratosMultiphysics as KM
+import numpy as np
+
+def Factory(settings, model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return AssignMPCsToNeighboursProcess(model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class AssignMPCsToNeighboursProcess(KM.Process):
```

Comment:
I'd call this `AssignMasterSlaveConstraintsToNeighboursProcess` to follow the naming convention of the rest of the Kratos (see the `ModelPart` for instance).